synapse/models/person.py

```diff
@@ -25,6 +25,8 @@ def getModelDefs(self):
                     'ex': 'robert grey'
                 }),
                 ('ps:person', ('guid', {}), {
+                    'interfaces': ('phys:object',),
+                    'template': {'phys:object': 'person'},
```

Comment:
Is the intent for the phys:object characteristics to represent the "most recent" height/weight/mass/etc? We already have ps:vitals.

Related comment, should ps:vitals inherit phys:object and have things like :height / :weight deprecated?

Reply:
Most recent and/or reported ( in the case of entity resolution ). But i def missed the `ps:vitals` and it should def inherit the `phys:object` interface. Good catch!